spec/views/devise/sessions/new.html.slim_spec.rb

```diff
@@ -39,17 +39,17 @@
     it 'displays a custom header' do
       render
 
-      expect(rendered).to have_content(t('headings.log_in_branded',
-                                         app: 'Awesome Application!'))
+      expect(rendered).to have_content(
+        t('headings.log_in_branded', sp: 'Awesome Application!')
+      )
     end
 
     it 'displays a back to sp link' do
       render
 
-      expect(rendered).
-        to have_link(
-          t('links.back_to_sp', app: 'Awesome Application!'), href: @sp_return_url
-        )
+      expect(rendered).to have_link(
+        t('links.back_to_sp', sp: 'Awesome Application!'), href: @sp_return_url
+      )
```

Comment:
ah ok. I realized through this that the test below actually had a misleading title. Updated and added specs for other side of conditional here e440c0d